src/audio/audio.cpp

```diff
@@ -486,16 +496,32 @@ void Audio::doCapture()
     }
 #endif
 
-    for (size_t i = 0; i < AUDIO_FRAME_SAMPLE_COUNT * AUDIO_CHANNELS; ++i)
-        buf[i] *= inGain;
+    for (quint32 i = 0; i < AUDIO_FRAME_SAMPLE_COUNT * AUDIO_CHANNELS; ++i)
+    {
+        // gain amplification with clipping to 16-bit boundaries
+        int ampPCM = qBound<int>(std::numeric_limits<int16_t>::min(),
+                                 qRound(buf[i] * gain),
+                                 std::numeric_limits<int16_t>::max());
+
+        buf[i] = static_cast<int16_t>(ampPCM);
+    }
 
     emit frameAvailable(buf, AUDIO_FRAME_SAMPLE_COUNT, AUDIO_CHANNELS, AUDIO_SAMPLE_RATE);
 }
 
 /**
+@internal
+*/
+void Audio::privateSetGain(qreal dB)
+{
+    gainDecibel = dB;
+    gain = qPow(10.0, (dB * 0.2));
```

Comment:
@initramfs Duh! Factor is wrong -> will be changed to **0.05**. Will add some graphs visualizing the (corrected) values. Stay tuned.